models/editVote.model.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+
+exports.editVote = (id, inc_votes) => {
+  return db.query(`SELECT votes FROM comments WHERE comment_id = $1;`, [id]).then(({ rows }) => {
+    const sum = rows[0].votes + inc_votes;
```

Comment:
Although this works you could refactor this to make the update to the votes using SQL only.  You would only need one query to do this (if you were being charged for each query to the DB, as can be the case for full production, this would be something to definitely consider) 👍